ci/scripts/util_enable_core_dumps.sh

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
```suggestion
#!/usr/bin/env bash
#
```

Reply:
No.

We should not add shebang to `util_*.sh` because we don't execute them directly. We use them as `source XXX.sh`.

Reply:
Thanks. I added `shellcheck disable=SC2148` instead of shebang.

Reply:
Can we use the following instead of disabling it?

https://www.shellcheck.net/wiki/SC2148

> Use a shellcheck [directive](https://www.shellcheck.net/wiki/directive), adding # shellcheck shell=ksh before the first command in the file.

Reply:
Thanks. replace to `shellcheck shell=bash`.